From 8178cc8403ca48ef72ee9da582bcea54f9e1ccf1 Mon Sep 17 00:00:00 2001
From: Zach Jang <zachjang@google.com>
Date: Mon, 8 Aug 2016 22:25:24 +0000
Subject: [PATCH 6/6] Revert "Fixed that emergency messages are not enabled for
 Japanese carriers"

This reverts commit bc0c1a983dd8d83d5c84afe4a2fa1f3ae09bed42.

Change-Id: Iec4f7fd440eaf9f2b3c1eb9587556098a98a195a
---
 .../android/cellbroadcastreceiver/CellBroadcastConfigService.java | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/src/com/android/cellbroadcastreceiver/CellBroadcastConfigService.java b/src/com/android/cellbroadcastreceiver/CellBroadcastConfigService.java
index a7273fb..451f9a7 100644
--- a/src/com/android/cellbroadcastreceiver/CellBroadcastConfigService.java
+++ b/src/com/android/cellbroadcastreceiver/CellBroadcastConfigService.java
@@ -344,18 +344,12 @@ public class CellBroadcastConfigService extends IntentService {
 
         /** Enable GSM ETWS series messages. */
 
-        // Enable/Disable GSM ETWS messages (4352 ~ 4354).
+        // Enable/Disable GSM ETWS messages.
         setCellBroadcastRange(manager, enableEtwsAlerts,
                 SmsManager.CELL_BROADCAST_RAN_TYPE_GSM,
                 SmsCbConstants.MESSAGE_ID_ETWS_EARTHQUAKE_WARNING,
                 SmsCbConstants.MESSAGE_ID_ETWS_EARTHQUAKE_AND_TSUNAMI_WARNING);
 
-        // Enable/Disable GSM ETWS other messages (4356).
-        setCellBroadcastRange(manager, enableEtwsAlerts,
-                SmsManager.CELL_BROADCAST_RAN_TYPE_GSM,
-                SmsCbConstants.MESSAGE_ID_ETWS_OTHER_EMERGENCY_TYPE,
-                SmsCbConstants.MESSAGE_ID_ETWS_OTHER_EMERGENCY_TYPE);
-
         // Enable/Disable GSM ETWS test messages (4355).
         setCellBroadcastRange(manager, enableEtwsTestAlerts,
                 SmsManager.CELL_BROADCAST_RAN_TYPE_GSM,
-- 
2.9.3

